deploy/runtools/run_farm.py

```diff
@@ -343,6 +343,21 @@ def load_edma(self):
         with StreamLogger('stdout'), StreamLogger('stderr'):
             run("sudo insmod /home/centos/edma/linux_kernel_drivers/edma/edma-drv.ko single_transaction_size=65536 transient_buffer_size=67108864 edma_queue_depth=1024 poll_mode=1")
 
+    def start_ila_server(self):
+        """ start the vivado hw_server and virtual jtag on simulation instance.) """
+        self.instance_logger("Starting Vivado hw_server.")
+        with StreamLogger('stdout'), StreamLogger('stderr'):
+            run("""screen -S hw_server -d -m bash -c "script -f -c '/opt/Xilinx/Vivado/2017.4.op/bin/hw_server'"; sleep 1""")
```

Comment:
I'm leery of hard coding the vivado version. Is hw_server not already on the path? If not we should vivado version stuff with the AFI name.